src/cephadm/cephadm

```diff
@@ -85,7 +85,8 @@ class Monitoring(object):
 
     port_map = {
         "prometheus": 9095,  # Avoid default 9090, due to conflict with cockpit UI
-        "node-exporter": 9100
+        "node-exporter": 9100,
+        "grafana": 3000
```

Comment:
```suggestion
        "grafana": 3000,
```

Reply:
updated the monitoring dicts to comply - and squashed